slackEvents.php

```diff
@@ -62,10 +62,6 @@ function app_home_page($userid, $filters_to_apply = array()) {
         foreach($events as $file=>$parsed_event) {
             $all_filters = array_merge($all_filters, $parsed_event["categories"]);
             
-            if($parsed_event["keep"] === false) {
```

Comment:
Could you help me understand why we can remove it?
(my understanding is that `Agenda.getUsersEventsFiltered` may still return events with `keep` set to false, so I'm a bit lost here)

Reply:
Indeed, eca38f0e9e0de43957a7b80e5ace1ff2b46c844e adds the missing condition.